Report/sections/2_design/Design-Choice

```diff
@@ -0,0 +1,10 @@
+
+\section{Design choice}
+
+
+The language is intended to be a transitional language from visual programming to high-level language. Based on the problem analysis, it has been possible to specify the design for the user so that it hits the target group.
+ The target group is beginners in programming who are younger people. Therefore a programming language to make computer games has been built. This has been chosen as young people can relate more to video games as their hobby. 
+ This will help them to understand the principles of how a game should work more easily, and therefore be able to start programming. 
+ It was decided to develop a language inspired on the game "the Legend of Zelda: Link’s Awakening" into the programming language, as it is a familiar game with basic game terminologies.
```

Comment:
"as it is a familiar game with basic game terminologies" 

måske ændre 'familiar' med 'well known'?